<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>35</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>35</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
   background:rgba(52,52,52,184);
}
QWidget#controlBar{
   background:rgba(52,52,52,184);
}
QSlider#timeSlider::groove:horizontal {
     border: 1px solid #999999;
     height:1px;
    /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: grey;
     
 }
QSlider#timeSlider::handle:horizontal {
     background: rgb(52,52,52);
     border: 1px solid #5c5c5c;
     width:6px;
     height:6px;
     
     margin: -8px 0px;  /*handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 3px;
 }
 QSlider#timeSlider::add-page:horizontal {
     background: grey;
 }

 QSlider#timeSlider::sub-page:horizontal {
     background: rgb(0, 137, 156);
     height:3px;
 }

/* volumeSlider */

QSlider#volumeSlider::groove:horizontal {
     border: 0px solid #999999;
     height:1px;
    /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: grey;
     
 }
QSlider#volumeSlider::handle:horizontal {
     background: rgb(52,52,52);
     border: 1px solid #5c5c5c;
     width:8px;
     height:8px;
     
     margin: -5px 0px;  /*handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 4px;
 }
 QSlider#volumeSlider::add-page:horizontal {
     background: grey;
 }

 QSlider#volumeSlider::sub-page:horizontal {
     background: rgb(0, 137, 156);
     height:3px;
 }
QCheckBox{color:rgba(255,0,0,0);}
QCheckBox::indicator{width:0px;height:0px;}
.QPushButton{border-style:hidden;background-repeat: no-repeat;background-position:center;}
QPushButton#slowBtn{background-image: url(:/img/img/slower2.png);}
QPushButton#slowBtn:hover{background-image: url(:/img/img/slower2_on.png);}
QPushButton#fastBtn{background-image: url(:/img/img/faster2.png);}
QPushButton#fastBtn:hover{background-image: url(:img/img/faster2_on.png);}


QCheckBox#stopBtn{background-image: url(:/img/img/icon7.png);}
QCheckBox#stopBtn:hover{background-image: url(:/img/img/icon7_on.png);}

QCheckBox#playBtn:unchecked {background-image: url(:/img/img/icon6.png);}
QCheckBox#playBtn:unchecked:hover{background-image: url(:/img/img/icon6_on.png);}
QCheckBox#playBtn:checked {background-image: url(:/img/img/icon10.png);}
QCheckBox#playBtn:checked:hover{background-image: url(:/img/img/icon10_on.png);}


QCheckBox#soundBtn:unchecked {background-image: url(:/img/img/icon8_1.png);
}
QCheckBox#soundBtn:unchecked:hover{background-image: url(:/img/img/icon8_1_on.png);
}
QCheckBox#soundBtn:checked {background-image: url(:/img/img/icon12.png);
}
QCheckBox#soundBtn:checked:hover{background-image: url(:/img/img/icon12_on.png);
}
QLabel
{
   color:rgb(160,160,160);
  
}
QLabel#curLabel{
   Alignment:right;
}
QLabel#totalLabel{
   Alignment:left;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="controlBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>530</width>
       <height>60</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>35</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <widget class="QCheckBox" name="playBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>5</y>
           <width>25</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>888</string>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
        <widget class="QCheckBox" name="stopBtn">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>5</y>
           <width>25</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>888</string>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="curLabel">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>5</y>
           <width>47</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="totalLabel">
         <property name="geometry">
          <rect>
           <x>97</x>
           <y>5</y>
           <width>48</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>111111</width>
          <height>35</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>530</width>
          <height>25</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="slowBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="timeSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>103</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>340</width>
               <height>20</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fastBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>18</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>35</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <widget class="QCheckBox" name="soundBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>5</y>
           <width>25</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>88</string>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
        <widget class="QSlider" name="volumeSlider">
         <property name="geometry">
          <rect>
           <x>23</x>
           <y>7</y>
           <width>50</width>
           <height>20</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mediaRes.qrc"/>
 </resources>
 <connections/>
</ui>
